### PR TITLE
Bagel song.

### PR DESCRIPTION
**Pull request by kneitinjan**

### CODE REVIEW

**Comment by marlaannhay** [comment]
I really enjoyed the third movement of that bagel.

File: 4minutes33bagels.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[4 Bagels 33"](https://www.youtube.com/watch?v=JTEFKFiXSx4)
+===================
```
Comment (Janrain-Colin) [inline 4minutes33bagels.md]: To comply with formatting in this repo, please make this line the same length as `[4 Bagels 33"]` and insert another line break after it.

Comment (kneitinjan) [inline 4minutes33bagels.md]: Thats not how post modernism works bub.

But I will concede.

**Comment by robotbill** [comment]
This is good, but I'd really like to see a rendition of 4'33 played on a bagel.

**Comment by kneitinjan** [comment]
@robotbill There a chorus of bagels performing it in the kitchen right now.

**Pull request merged**